tests/components/dlna_dmr/conftest.py

```diff
@@ -11,17 +11,18 @@
 
 from homeassistant.components.dlna_dmr.const import DOMAIN as DLNA_DOMAIN
 from homeassistant.components.dlna_dmr.data import DlnaDmrData
-from homeassistant.const import CONF_DEVICE_ID, CONF_TYPE, CONF_URL
+from homeassistant.const import CONF_DEVICE_ID, CONF_MAC, CONF_TYPE, CONF_URL
 from homeassistant.core import HomeAssistant
 
 from tests.common import MockConfigEntry
 
-MOCK_DEVICE_BASE_URL = "http://192.88.99.4"
-MOCK_DEVICE_LOCATION = MOCK_DEVICE_BASE_URL + "/dmr_description.xml"
+MOCK_DEVICE_HOST_ADDR = "192.88.99.4"
```

Comment:
Any reason this is a public IP? I'd expect most devices to have a private/local IP such as 192.168....

Reply:
I thought it was meant to be a documentation/example IP range, but it turns out I was mistaken, as specified in [RFC5737](https://datatracker.ietf.org/doc/html/rfc5737). The original reason was that if I'd failed to mock outbound network connections correctly, at least it wouldn't go to any real hosts.

I can put in a separate PR, or update this to fix it?

Reply:
I've updated this PR to fix it.